qpext/cmd/qpext/main.go

```diff
@@ -192,6 +239,7 @@ func (sc *ScrapeConfigurations) handleStats(w http.ResponseWriter, r *http.Reque
 	w.Header().Set("Content-Type", string(format))
 
 	// Write out the metrics
+	//TODO: do we need this?
```

Comment:
we originally took inspo from istio - but when I tested, this doesn't do anything? its an empty []